docs/bin/plugin_formatter.py

```diff
@@ -247,6 +259,12 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
         # use ansible core library to parse out doc metadata YAML and plaintext examples
         doc, examples, returndocs, metadata = plugin_docs.get_docstring(module_path, fragment_loader, verbose=verbose)
 
+        for key, opt in (doc.get('options') or {}).items():
+            try:
+                doc['options'][key] = to_boolean(opt)
+            except TypeError:
+                pass
```

Comment:
Ah...  I think I see what you are doing... (About None)

You're trying on convert_bool.boolean to raise a TypeError if it's not one of the recognized values....

If you're intentionally using that strategy, do the following:

* In the call to Boolean(), explicitly set strict=True.  That way everyone knows that you're telling on the TypeError and your protected in case the default changes in the future.
* Move this check into your normalization function.  It's part of determining what the value of the key should be normalized to, so it belongs with the other normalization code, not out at the caller where it looks like error handling.

Reply:
I'm not sure about the second point: TypeError explicitly tells the caller that value wasn't boolean, while if it would be hidden inside of to_boolean it would imply that boolean conversion succeeded.
We could do another function `try_convert_to_boolean()`, as would be better understandable semantically what it does.

Reply:
but to_boolean doesn't convert values to booleans.  It's really normalize_options().  It's just that the only implemented normalization at the moment is for boolean values.

Reply:
Yea, so we could go for changing naming

Reply:
OK, rename the function and move the check

Reply:
@abadger modified the structure, could you please review again?